contracts/zero-ex/contracts/src/features/Ownable.sol

```diff
@@ -34,26 +35,36 @@ contract Ownable is
     IOwnable,
     FixinOwnable
 {
-    // solhint-disable const-name-snakecase
 
+    // solhint-disable const-name-snakecase
     /// @dev Name of this feature.
     string constant public override FEATURE_NAME = "Ownable";
     /// @dev Version of this feature.
     uint256 constant public override FEATURE_VERSION = (1 << 64) | (0 << 32) | (0);
+    // solhint-enable const-name-snakecase
+
+    // solhint-disable
+    /// @dev The deployed address of this contract.
+    address immutable private _implementation;
+    // solhint-enable
+
+    constructor() public {
+        _implementation = address(this);
+    }
```

Comment:
Also switched to this pattern instead of explicitly passing in `impl` in `bootstrap()`. This is the pattern we will also follow for `migrate()`.